src/Core/HttpClients/CurlHttpClient.php

```diff
@@ -62,7 +62,7 @@ public function prepareRequest($url, $method, array $headers, $body, $timeOut, $
             CURLOPT_RETURNTRANSFER => true,
             CURLOPT_HTTPHEADER => $this->getHeaders($headers),
             //10 seconds is allowed to make the connection to the server
-            CURLOPT_CONNECTTIMEOUT => 10,
+            CURLOPT_CONNECTTIMEOUT => isset($timeOut) ? $timeOut : 100,
```

Comment:
I think 10 is too low and change it to at least 15 (so it's half of the default PHP max_execution_time of 30).
Also I think it would be nice if there was a way to globally overwrite the default value, maybe using a constant

Reply:
Added QUICKBOOKS_API_TIMEOUT and set default to 15